packages/fiori/test/specs/Wizard.spec.js

```diff
@@ -36,7 +36,7 @@ describe("Wizard general interaction", () => {
 			"Wizard nav has aria-label set.");
 		assert.strictEqual(wizList.getAttribute("role"), "list",
 			"Wizard list has role set..");
-		assert.strictEqual(wizList.getAttribute("aria-controls"), "ui5-wiz-content",
+		assert.ok(wizList.getAttribute("aria-controls"),
```

Comment:
```assert.strictEqual(`${wizList.getAttribute("id")}${wizList.getAttribute("aria-controls")}`)```

This way we check not only if there is a string, but if the string is the correct one.

Reply:
Done.